_data/glossary/docker.yaml

```diff
@@ -0,0 +1,9 @@
+id: docker
+name: docker
+full-link: /docs/reference/kubectl/docker-cli-to-kubectl/
+tags:
+- fundamental
+short-description: >
+  Docker is a software technology providing operating-system-level virtualization also known as containers.
+long-description: >
+  Docker uses the resource isolation features of the Linux kernel such as cgroups and kernel namespaces, and a union-capable file system such as OverlayFS and others to allow independent "containers" to run within a single Linux instance, avoiding the overhead of starting and maintaining virtual machines (VMs).
```

Comment:
"kernel namespaces" in this context is a little bit too wordy, we can remove "kernel" here.
"union-capable filesystem such as OverlayFS and others" can be removed as well because it reads like such a file system is mandatory. In fact, there are block-level drivers for docker storage.